CMakeLists.txt

```diff
@@ -238,13 +238,16 @@ if(ASAN)
   target_link_libraries(devilution PUBLIC -fsanitize=address -fsanitize-recover=address)
 endif()
 
-if(DIST)
+if(DIST AND NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
   target_link_libraries(devilutionx PUBLIC -static-libgcc -static-libstdc++)

```

Comment:
Is there a MSVC equvilant to this?

Reply:
Looks like it would be `/MT`
https://docs.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=vs-2019

Reply:
Yes that is correct, however you need to pass `/MTd` in debug which is very annoying to do in cmake. Also mixing `/MT` with `/MD` is instant linking error so it's better to be careful with this. I can look into it some time in the future if you wish.

Reply:
Or maybe this can be skipped altogether. There aren't a lot of good reasons to statically link libc/libc++. Binary compatibility is pretty much guaranteed on Windows. The worst case scenario would be having to install certain version of `msvcrt`.

Reply:
Actually that's true, in Windows 10 there should be no problem at all, on other Windows versions it generally one redistributable/update away from working so in general it's not worth it now.

Reply:
Since this is only for DIST we woudn't need to care about /MTd

Reply:
Modern MSVC links against a university C runtime which is updated in a backwardly compatible fashion and should work like the msvcrt.dll of old did where you could rely on it being present. Static linking the runtime is discouraged by microsoft.

Reply:
This makes sense. Maybe leave a comment over at https://github.com/diasurgical/devilutionX/issues/146 so that it can be discussed there.